js/main.js

```diff
@@ -14,17 +14,40 @@ function addingInputIntoObject(event) {
   var $title = document.querySelector('.title-value');
   var $photoURL = document.querySelector('#photo-URL');
   var $notes = document.querySelector('.notes-value');
+  var $imagePlaceHolder = document.querySelector('.image-place-holder');
   var inputs = {};
   inputs.title = $title.value;
   inputs.photoURL = $photoURL.value;
   inputs.notes = $notes.value;
-  inputs.entryid = data.nextEntryId;
-  data.nextEntryId++;
-  data.entries.unshift(inputs);
+  if ($imagePlaceHolder.matches('[data-entry-id]')) {
+    inputs.entryid = $imagePlaceHolder.getAttribute('data-entry-id');
+    inputs.entryid = parseInt(inputs.entryid);
+    for (var dataEntryNumber = 1; dataEntryNumber <= data.entries.length; dataEntryNumber++) {
```

Comment:
You should just start `dataEntryNumber` at `0` so you don't need to subtract 1 from it.